zyte_spider_templates/spiders/base.py

```diff
@@ -49,6 +49,10 @@ class BaseSpiderParams(BaseModel):
             "widget": "request-limit",
         },
     )
+    allow_items_outside_domains: Optional[bool] = Field(
+        description="When set to True, items outside of the domains will be crawled.",
+        default=False,
```

Comment:
Do you think it's feasible NOT to add this option, but make it a default instead? It looks safe enough, at least for "navigation" and "pagination_only" strategies. Maybe it's fine for "full" as well. We might need to do some experiment to ensure it's safe though.

Reply:
After some analysis, it turns out we can enabled crawling products outside of the domain by default. I guess the ultimate switch by users here would be enabling the `AllowOffsiteMiddleware` in settings.